src/components/Navbar/navbar.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import '../../css/navbar.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faHome,
+  faClipboard,
+  faSignOutAlt,
+  faCalendarCheck,
+  faCar,
+  faTrashAlt,
+} from '@fortawesome/free-solid-svg-icons';
+import { openForm } from '../../redux/car_list/addNewCarFormSlice';
+
+const Navbar = (props) => {
+  const dispatch = useDispatch();
+  const { Page } = props;
+  const openFormHandle = () => {
+    dispatch(openForm());
+  };
+  return (
+    <div className="container-fluid gap-0">
+      <div className="row flex-nowrap">
+        <div className="col-auto col-md-3 col-xl-2 ps-sm-2 px-0 bg-white">
+          <div className="border-right d-flex flex-column align-items-center align-items-sm-start px-0 pt-2 text-white min-vh-100">
+            <NavLink
+              exact
+              to="/"
+              className="d-flex align-items-center pb-3 mb-md-0 me-md-auto text-white text-decoration-none"
+            >
+              <span className="fs-5 d-none d-sm-inline">Menu</span>
+            </NavLink>
+            <ul
+              className="w-100 nav  flex-column mb-sm-auto mb-0 align-items-center align-items-sm-start"
+              id="menu"
+            >
+              <li className="px-1 nav-item w-100 hoverli">
+                <NavLink
+                  to="/"
+                  className={(isActive) => `hoverlink  align-middle px-0${!isActive ? ' selected' : ''}`}
```

Comment:
- [ ] Since you are using this function inside `className` a lot, why don't you create a function and just call it where you need?